docs/DeveloperGuide.md

```diff
@@ -90,22 +90,23 @@ Here's a (partial) class diagram of the `Logic` component:
 
 <puml src="diagrams/LogicClassDiagram.puml" width="550"/>
 
-The sequence diagram below illustrates the interactions within the `Logic` component, taking `execute("delete 1")` API call as an example.
+The sequence diagram below illustrates the interactions within the `Logic` component, taking `execute("export")` API call as an example.
 
-<puml src="diagrams/DeleteSequenceDiagram.puml" alt="Interactions Inside the Logic Component for the `delete 1` Command" />
+<puml src="diagrams/ExportSequenceDiagram.puml" alt="Interactions Inside the Logic Component for the `export` Command" />
+
+How the `Logic` component works:
+
+1. When `Logic` is called upon to execute a command, it is passed to an `AddressBookParser` object which in turn creates a parser that matches the command (e.g., `DeleteCommandParser`) and uses it to parse the command.
 
 <box type="info" seamless>
 
-**Note:** The lifeline for `DeleteCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline continues till the end of diagram.
+**Note:** For the `export` command above, a parser that matches the command is not created by `AddressBookParser` as it only contains a command and does not contain an argument. However, for other relevant commands, `AddressBookParser` will create the parser that matches the command.
```

Comment:
Good clarification on the export command diagram not using an export command parser